test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectConfigurationTests.cs

```diff
@@ -221,5 +220,18 @@ public void RoundTripFromJson()
 
             TestUtilities.AssertFailIfErrors(context);
         }
+
+        [Fact]
+        public void ConditionalCollectionSerialization()
+        {
+            var context = new CompareContext {Title = "ConditionalCollectionSerialization"};
+            // Initialize an OpenIdConnectConfiguration object with all collections empty.
+            var oidcWithEmptyCollections = new
```

Comment:
Can you put this on one line.

Reply:
Done! Not sure why this wasn't on one line in the first place. :confused: